content/cookbook/development/using-mainnet-accounts-programs.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Using Mainnet Accounts and Programs
+sidebarSortOrder: 5
+description:
+  Learn how to use the Mainnet accounts and programs in your local development
+  environment.
+---
+
+Oftentimes, local tests rely on programs and accounts available only on mainnet.
+The Solana CLI allows to both:
+
+- Download Programs and Accounts
+- Load Programs and Accounts to a local validator
+
+### How to load accounts from mainnet
+
+It is possible to download the SRM token mint account to file:
+
+```shell
+# solana account -u <source cluster> --output <output format> --output-file <destination file name/path> <address of account to fetch>
+solana account -u m --output json-compact --output-file jup.json JUPyiwrYJFskUPiHa7hkeR8VUtAeFoSYbKedZNsDvCN
```

Comment:
That's the Jupiter address. Nothing to do with `SRM`. Maybe this got messed up in editing?

Reply:
YESSER Thanks